modules/ROOT/pages/configure-acm.adoc

```diff
@@ -427,10 +420,12 @@ image::access-mapping.png[Access Mapping for CommunityApi]
 ... *AnypointContracts*
 .. Enable *Read* and *Edit* permissions for:
 ... *SelfRegisterUserRequests*
-
+. Select *Save*.
```

Comment:
Select from a list or click an icon or button?

Reply:
This should be "click" and many other instances of "select" should be too. I used find and replace to fix this.